eng/scripts/get-npm-tags-for-version.ps1

```diff
@@ -0,0 +1,20 @@
+param (
+  [Parameter(mandatory = $true)]
+  $packageArtifact,
+  [Parameter(mandatory = $true)]
+  $workingDirectory
+)
+
+$HelpersPath = Join-Path $PSScriptRoot "helpers"
+. (Join-Path $HelpersPath npm-helpers.ps1)
+
+$pkgProps = Get-javascript-PackageInfoFromPackageFile -pkg $packageArtifact -workingDirectory $workingDirectory
+if ($pkgProps -eq $null)
+{
+  Write-Error "Failed to parse package artifact $packageArtifact to get package name"
+  exit 1
+}
+$packageName = $pkgProps.PackageId
+$packageVersion = $pkgProps.PackageVersion
+$result = GetCurrentNpmTags -packageName $packageName -packageVersion $packageVersion
```

Comment:
Should we just add in a new parameter for get-npm-tags.ps1 instead of having two scripts?

Reply:
Unless I'm missing something it seems like this is the only line that is different between https://github.com/Azure/azure-sdk-for-js/blob/main/eng/scripts/get-npm-tags.ps1. If that is true then I do agree with Albert that we should just add a parameter that would toggle which tags to get.

Reply:
Removed duplicate function